features/admin.claims.v1/components/wizard/local-claim/basic-details-local-claims.tsx

```diff
@@ -221,8 +221,8 @@ export const BasicDetailsLocalClaims = (props: BasicDetailsLocalClaimsPropsInter
                                         loading={ validateMapping }
                                         listen={ (values: Map<string, FormValue>) => {
                                             setClaimID(values.get("claimURI").toString());
-                                            setOidcMapping(values.get("claimURI").toString().replace(/\./g,""));
-                                            setScimMapping(values.get("claimURI").toString().replace(/\./g,""));
+                                            setOidcMapping(values.get("claimURI").toString());
```

Comment:
Seems we have replaced dots as a fix for some issue. Hope this is no longer relevant https://github.com/wso2/identity-apps/pull/2310

cc: @brionmario

Reply:
It was added in this PR.
https://github.com/wso2/identity-apps/pull/2310

Not reason was mentioned.
How ever having dot is required in mappings as well. Hence will proceed with this.